README.md

```diff
@@ -18,6 +18,10 @@ is installing the client on your workstation.
 
 ## Installation
 
+macOS users can install using `brew`
```

Comment:
I think we definitely want to add this to the instructions, but it probably also needs to go with some sort of restructuring of this section of the doc to make the separation of topics more clear.

I'll spend a bit of time thinking about that and then probably merge this change immediately followed by a change to restructure this section.